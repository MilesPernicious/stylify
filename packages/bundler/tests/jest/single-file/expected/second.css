#stylify-profiler .hover\:background\:\#333:hover,
#stylify-profiler .profiler-extension:hover{
	background: #333
}
#stylify-profiler .border-top\:1px__solid__\#444{
	border-top: 1px solid #444
}
#stylify-profiler .background-color\:\#333,
#stylify-profiler .profiler-extension__button--active{
	background-color: #333
}
#stylify-profiler .bottom\:0{
	bottom: 0
}
#stylify-profiler .color\:\#fff{
	color: #fff
}
#stylify-profiler .content-visibility\:hidden{
	content-visibility: hidden
}
#stylify-profiler .content-visibility\:visible{
	content-visibility: visible
}
#stylify-profiler .display\:block{
	display: block
}
#stylify-profiler .display\:none{
	display: none
}
#stylify-profiler .font-family\:arial{
	font-family: arial
}
#stylify-profiler .font-size\:12px,
#stylify-profiler .profiler-extension__button{
	font-size: 12px
}
#stylify-profiler .line-height\:20px{
	line-height: 20px
}
#stylify-profiler .margin-left\:8px{
	margin-left: 8px
}
#stylify-profiler .margin\:0__8px{
	margin: 0 8px
}
#stylify-profiler .max-width\:800px{
	max-width: 800px
}
#stylify-profiler .overflow-x\:auto{
	overflow-x: auto
}
#stylify-profiler .padding-top\:8px{
	padding-top: 8px
}
#stylify-profiler .position\:fixed{
	position: fixed
}
#stylify-profiler .text-align\:left{
	text-align: left
}
#stylify-profiler .visibility\:hidden{
	visibility: hidden
}
#stylify-profiler .visibility\:visible{
	visibility: visible
}
#stylify-profiler .width\:auto{
	width: auto
}
#stylify-profiler .word-spacing\:24px{
	word-spacing: 24px
}
#stylify-profiler .box-sizing\:border-box,
#stylify-profiler .profiler-extension,
#stylify-profiler .profiler-extension__button,
#stylify-profiler .profiler-extension__dropdown,
#stylify-profiler .profiler-extension__link{
	box-sizing: border-box
}
#stylify-profiler .border-left\:1px__solid__\#555,
#stylify-profiler .profiler-extension{
	border-left: 1px solid #555
}
#stylify-profiler .align-items\:center,
#stylify-profiler .profiler-extension,
#stylify-profiler .profiler-extension__button{
	align-items: center
}
#stylify-profiler .display\:flex,
#stylify-profiler .profiler-extension,
#stylify-profiler .profiler-extension__button{
	display: flex
}
#stylify-profiler .min-height\:100\%,
#stylify-profiler .profiler-extension{
	min-height: 100%
}
#stylify-profiler .position\:relative,
#stylify-profiler .profiler-extension{
	position: relative
}
#stylify-profiler .height\:20px,
#stylify-profiler .profiler-extension__button{
	height: 20px
}
#stylify-profiler .padding\:0__8px,
#stylify-profiler .profiler-extension__button{
	padding: 0 8px
}
#stylify-profiler .justify-content\:center,
#stylify-profiler .profiler-extension__button{
	justify-content: center
}
#stylify-profiler .cursor\:pointer,
#stylify-profiler .profiler-extension__button,
#stylify-profiler .profiler-extension__link{
	cursor: pointer
}
#stylify-profiler .user-select\:none,
#stylify-profiler .profiler-extension__button{
	user-select: none
}
#stylify-profiler .position\:absolute,
#stylify-profiler .profiler-extension__dropdown{
	position: absolute
}
#stylify-profiler .bottom\:100\%,
#stylify-profiler .profiler-extension__dropdown{
	bottom: 100%
}
#stylify-profiler .left\:0,
#stylify-profiler .profiler-extension__dropdown{
	left: 0
}
#stylify-profiler .max-height\:50vh,
#stylify-profiler .profiler-extension__dropdown{
	max-height: 50vh
}
#stylify-profiler .overflow\:auto,
#stylify-profiler .profiler-extension__dropdown{
	overflow: auto
}
#stylify-profiler .min-width\:100\%,
#stylify-profiler .profiler-extension__dropdown{
	min-width: 100%
}
#stylify-profiler .background\:\#000,
#stylify-profiler .profiler-extension__dropdown{
	background: #000
}
#stylify-profiler .padding\:8px,
#stylify-profiler .profiler-extension__dropdown{
	padding: 8px
}
#stylify-profiler .text-decoration\:none,
#stylify-profiler .profiler-extension__link{
	text-decoration: none
}
#stylify-profiler .color\:\#00b2e5,
#stylify-profiler .profiler-extension__link{
	color: #00b2e5
}
#stylify-profiler .margin-right\:8px,
#stylify-profiler .profiler-extension__link,
#stylify-profiler .profiler-extension__button-icon{
	margin-right: 8px
}
#stylify-profiler .white-space\:nowrap,
#stylify-profiler .profiler-extension__link{
	white-space: nowrap
}
#stylify-profiler .display\:inline-block,
#stylify-profiler .profiler-extension__link,
#stylify-profiler .profiler-extension__button-icon{
	display: inline-block
}
#stylify-profiler .width\:12px,
#stylify-profiler .profiler-extension__button-icon{
	width: 12px
}
#stylify-profiler .height\:12px,
#stylify-profiler .profiler-extension__button-icon{
	height: 12px
}
#stylify-profiler .font-weight\:bold,
#stylify-profiler .profiler-extension__button-icon{
	font-weight: bold
}
#stylify-profiler .color\:\#aaa,
#stylify-profiler .profiler-extension__button-icon{
	color: #aaa
}
#stylify-profiler .line-height\:1,
#stylify-profiler .profiler-extension__button-label{
	line-height: 1
}
