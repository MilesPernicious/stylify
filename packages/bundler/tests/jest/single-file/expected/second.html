<!--
	@stylify-components
		'profiler-extension': `
			box-sizing:border-box
			border-left:1px__solid__#555
			align-items:center
			display:flex
			min-height:100%
			position:relative
			hover:background:#333
		`,
		'profiler-extension__button': `
			box-sizing:border-box
			height:20px
			padding:0__8px
			display:flex
			align-items:center
			justify-content:center
			font-size:12px
			cursor:pointer
			user-select:none
		`,
		'profiler-extension__button--active': 'background-color:#333',
		'profiler-extension__dropdown': `
			box-sizing:border-box
			position:absolute
			bottom:100%
			left:0
			max-height:50vh
			overflow:auto
			min-width:100%
			background:#000
			padding:8px
		`,
		'profiler-extension__link': `
			box-sizing:border-box
			text-decoration:none
			color:#00b2e5
			margin-right:8px
			white-space:nowrap
			display:inline-block
			cursor:pointer
		`,
		'profiler-extension__button-icon': `
			width:12px
			height:12px
			margin-right:8px
			display:inline-block
			font-weight:bold
			color:#aaa
		`,
		'profiler-extension__button-label': 'line-height:1'
	/@stylify-components

	@stylify-pregenerate
		border-top:1px__solid__#444
		background-color:#333
		bottom:0
		color:#fff
		content-visibility:hidden
		content-visibility:visible
		display:block
		display:none
		font-family:arial
		font-size:12px
		line-height:20px
		margin-left:8px
		margin:0__8px
		max-width:800px
		overflow-x:auto
		padding-top:8px
		position:fixed
		text-align:left
		visibility:hidden
		visibility:visible
		width:auto
		word-spacing:24px
		profiler-extension
		profiler-extension__button
		profiler-extension__button--active
		profiler-extension__dropdown
		profiler-extension__link
		profiler-extension__button-icon
		profiler-extension__button-label
	/@stylify-pregenerate
-->
